projects/i18n/src/modules/i18n/get-string-for-locale.ts

```diff
@@ -8,19 +8,21 @@ import {
  */
 export function getStringForLocale(
   resources: {
-    [locale: string]: SkyAppResources
+    [locale: string]: {
+      [name: string]: string;
+    }
   },
   preferredLocale: string,
   name: string
-): string {
+): string | undefined {
   const defaultLocale = 'en-US';
 
-  function getResourcesForLocale(locale: string): SkyAppResources {
+  function getResourcesForLocale(locale: string): {[name: string]: string} {
     const parsedLocale = locale.toUpperCase().replace('_', '-');
     return resources[parsedLocale];
   }
 
-  let values: any = getResourcesForLocale(preferredLocale);
+  let values: {[name: string]: string} = getResourcesForLocale(preferredLocale);
```

Comment:
Because this was cast to `any` it was swallowing an actual error. The builder plugin is passing `{[name: string]: string}` to the `getStringForLocale` function, _NOT_ `SkyAppResources`.
See: https://github.com/blackbaud/skyux-sdk-builder-plugin-skyux/blob/master/src/resources-provider.js#L77